modules/troubleshooting-network-observability-flowlogs-pipeline-kafka.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="configure-network-traffic-flowlogs-pipeline-kafka_{context}"]
+= Configuring flowlogs-pipeline to consume network flow from Kafka
+
+Manually restart the flow-pipeline pods where Flowlogs-pipeline does not consume network flow from Kafka.
+
+.Procedure
+
+. Delete the flow-pipeline pods to restart them by running the following command:
++
+[source,terminal]
+----
+$ oc -n netobserv delete -f ./path/to/your/flowcollector.yaml
+----
+
+. Create the `flowcollector` object by running the following command:
++
+[source,terminal]
+----
+$ oc -n netobserv apply -f ./path/to/your/flowcollector.yaml
+----
```

Comment:
To restart the FLPs, you only need to run `oc delete pods -n netobserv -l app=flowlogs-pipeline-transformer`

Reply:
Addressed